tests/test_ampOffset.py

```diff
@@ -40,91 +47,362 @@ def setUp(self):
+                    -1.63924511,
+                    -0.16414238,
+                    -0.54299292,
+                    0.30466945,
+                    0.25016478,
+                    -0.77477473,
+                    1.25528757,
+                ],
+                "artificial": [
+                    -3.93099297,
+                    -6.63242338,
+                    -4.86026077,
+                    -9.57856454,
+                    8.68414419,
+                    5.6553814,
+                    3.5965008,
+                    7.06621528,
+                ],
+            },
+        }
+        self.measuredSigma = {
+            "unweighted": {
+                "symmetric": 3.550909965491515e-08,
+                "random": 9.271480035637249e-08,
+                "artificial": 3.9430895759341275e-14,
+            },
+            "weighted": {
+                "symmetric": 3.6093886632617167e-08,
+                "random": 9.280982569622311e-08,
+                "artificial": 8.690997162288794e-15,
+            },
+        }
+        self.measuredSigmaConstantBackground = {
+            "unweighted": {
+                "symmetric": 0.06679099233901276,
+                "random": 0.12836424357658588,
+                "artificial": 0.5766923710558869,
+            },
+            "weighted": {
+                "symmetric": 0.06679099233983563,
+                "random": 0.12854047710163782,
+                "artificial": 0.579062202931159,
+            },
+        }
+        self.measuredSigmaRampBackground = {
+            "unweighted": {
+                "symmetric": 0.8049174282700369,
+                "random": 0.8128787501698868,
+                "artificial": 6.090700238137678,
+            },
+            "weighted": {
+                "symmetric": 0.8049174282702547,
+                "random": 0.8089364786360683,
+                "artificial": 6.089900455983161,
+            },
+        }
 
     def tearDown(self):
         del self.mock
 
-    def buildExposure(self, addBackground=False):
+    def buildExposure(self, valueType, addBackground=False, rampBackground=False):
+        """
+        Build and return an exposure with different types of value
+        distributions across its amplifiers.
+
+        Parameters
+        ----------
+        valueType : `str`
+            Determines the distribution type of values across the amplifiers.
+            - "symmetric": Creates a symmetric constant interval distribution
+            of values.
+            - "random": Generates a random distribution of values.
+            - "artificial": Uses a predefined array of values to simulate a
+            weight-sensitive condition for the output pedestals. This set of
+            values designed to show more change across the short interface and
+            will not solve exactly.
+
+        addBackground : `bool`, optional
+            If True, adds a background value to the entire exposure. Default is
+            False.
+
```

Comment:
It's usually advisable to _not_ put default values into doc strings, as they are usually not modified down the line when the defaults change. Perhaps it's ok here, as this is a unit test and so less likely to be modified, but you may want to consider changing it.